design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -11,22 +11,134 @@ type CartQueryOutput {
 }
 
 type Cart {
+    id: ID! @doc(description: "The ID of the cart.")
+    items: [CartItemInterface]
+    applied_coupons: [AppliedCoupon] @doc(description:"An array of `AppliedCoupon` objects. Each object contains the `code` text attribute, which specifies the coupon code")
```

Comment:
right now Magento supports only one coupon per cart, it is better to explicitly describe it in a comment.

Reply:
> right now Magento supports only one coupon per cart, it is better to explicitly describe it in a comment.


If we don't support this functionality, what's the value in having the functionality reflected in the schema at all>

Reply:
@DrewML This is already released schema and done this way to allow for future or 3rd party extensions. When support for multiple coupons is added clients will not have to be changed.

Reply:
Cool, if it's already out there let's just go with it 👍